src/ExpenseTrackerGrupo2/DataAccess/Entities/User.cs

```diff
@@ -0,0 +1,40 @@
+namespace ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Entities;
+
+public class User
+{
+    private Guid _id;
+    private string _name;
```

Comment:
cant be a null name, its needed to be setted like 'required', there are more casses of this in the PR.

Reply:
below it is requested to be required, such as:

```
public required Guid Id
{
   get => _id;
   set => _id = value;
}
```